_sass/_reset.scss

```
@@ -81,15 +75,10 @@ html {
 }
 
 %focus {
-  outline: 1px dotted #212121;
-  outline: 5px auto -webkit-focus-ring-color;
+    outline: 2px solid #177960;
```

```#177960``` is defined as a SASS variable: ```$link``` in _reset.scss
```suggestion
    outline: 2px solid $link;
```

---

@petetalksweb I agree completely. I had these same thoughts, but didn't want to touch the CSS structure too heavily on my first PR to the project. So, any feedback on which approach is best would be great! Happy to update the PR with either solution to avoid the hard-coded HEX value.

---

@davidluhr Fair points. Based on my understanding of the goals of a reset CSS file (based entirely on [this post I just read](https://cssreset.com/what-is-a-css-reset/)), it would make sense to me to move the ```%focus``` and ```a:focus``` styling from _reset.scss to _base.scss, and change the order of imports in master.scss to:
```
@import "reset";
@import "colors";
@import "base";
```
But I'm not a member of the project and have mostly just fixed capitalization errors on it. I'm sure @scottaohara and/or @ericwbailey will have thoughts.

---

@petetalksweb I agree completely. I think moving the focus mixin is the best solution, along with adjusting the import order for the variables file `colors.scss`. I'll defer to the project members, but thank you for all the feedback!

---

I agree on the mixin front. I personally don't use placeholders exactly because of these kinds of concerns.

---

Also in agreement over here.  Thank you all for taking the time to dive into this.